runtime-attach/src/main/java/io/opentelemetry/contrib/attach/RuntimeAttach.java

```diff
@@ -7,22 +7,99 @@
 
 import java.io.File;
 import java.lang.management.ManagementFactory;
+import java.util.logging.Logger;
 import net.bytebuddy.agent.ByteBuddyAgent;
 
+/** This class allows you to attach the OpenTelemetry Java agent at runtime. */
 public final class RuntimeAttach {
 
+  private static final Logger logger = Logger.getLogger(RuntimeAttach.class.getName());
```

Comment:
I might be a little bit late here, but... 😄 

Should we use JUL here? This class is supposed to be the very first thing called in `main`, right? JUL won't be set up this early, and these calls certainly won't be redirected to agent's `PatchLogger`. Wouldn't it be safer to use stderr here?

Reply:
> Should we use JUL here? This class is supposed to be the very first thing called in main, right? JUL won't be set up this early, and these calls certainly won't be redirected to agent's PatchLogger. Wouldn't it be safer to use stderr here?

Probably! 

What about the logging added by a user? For example, if a user has a  Spring Boot application and adds an import related to logging (jul, Log4j, logback, ...) in the class annotated with `@SpringBootApplication`, it would also  have a problem? Right?

```java
...
import java.util.logging.Logger;
...
@SpringBootApplication
public class SpringBootApp {
    public static void main(String[] args) {
        RuntimeAttach.attachJavaagentToCurrentJVM();
        SpringApplication.run(SpringBootApp.class, args);
    }
}
```
@mateuszrzeszutek @trask What do you think?

Reply:
thx, I agree with changing the logging in the runtime attach code itself to use stderr so that we don't trigger loading of j.u.l too early, e.g. in case the user is using jboss-logging (maybe https://www.wildfly.org/news/2020/06/18/Bootable-jar-Wildfly-20/?)

I don't think we need to add any restrictions on the user's logging setup. If whatever they were doing before they added the "runtime attach snippet" worked, it should still work after they add the snippet.